src/SFA.DAS.Authorization.EmployerFeatures/AuthorizationContextKeys.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.Authorization.EmployerFeatures
+{
+    internal static class AuthorizationContextKeys
+    {
+        public const string AccountId = "AccountId";
```

Comment:
As there is a single AuthorizationContext per set of handlers, then don't we need to prefix the key names to avoid name collisions?

Reply:
We could add a public `AuthorizationContextKeys` class to the `SFA.DAS.Authorization` package that contains the "shared" keys. It does mean that it would have knowledge of what keys the satellite packages need though?